extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/discovery/WemoLinkDiscoveryService.java

```diff
@@ -135,7 +142,14 @@ protected void startScan() {
                                 "</DeviceLists>");
 
                         stringParser = StringEscapeUtils.unescapeXml(stringParser);
+                        
+                        // check if there are already paired devices with WeMo Link
+                        if("0".equals(stringParser)) {
+                        	logger.info("There are no devices connected with WeMo Link. Exit discovery");
```

Comment:
Without the "return" statement a SAXParserException occurs every 120 seconds, if the discovery background job is started. I think it's a good idea keep the logfile clean.